contrib/index.html

```diff
@@ -0,0 +1,179 @@
+              <textarea id="text_area"></textarea></p>
+
+            <p><label for="select_element">Select Element:</label>
+              <select name="select_element">
+                <optgroup label="Option Group 1">
+                  <option value="1">Option 1</option>
+                  <option value="2">Option 2</option>
+                  <option value="3">Option 3</option>
+                </optgroup>
+                <optgroup label="Option Group 2">
+                  <option value="1">Option 1</option>
+                  <option value="2">Option 2</option>
+                  <option value="3">Option 3</option>
+                </optgroup>
+            </select></p>
+
+            <p><label for="radio_buttons">Radio Buttons:</label>
+              <label>
+                <input type="radio" class="radio" name="radio_button" value="radio_1" /> Radio 1
+              </label>
+              <label>
+                <input type="radio" class="radio" name="radio_button" value="radio_2" /> Radio 2
+              </label>
+              <label>
+                <input type="radio" class="radio" name="radio_button" value="radio_3" /> Radio 3
+              </label>
+            </p>
+
+            <p><label for="checkboxes">Checkboxes:</label>
+              <label>
+                <input type="checkbox" class="checkbox" name="checkboxes" value="check_1" /> Checkbox 1
+              </label>
+              <label>
+                <input type="checkbox" class="checkbox" name="checkboxes" value="check_2" /> Checkbox 2
+              </label>
+              <label>
+                <input type="checkbox" class="checkbox" name="checkboxes" value="check_3" /> Checkbox 3
+              </label>
+            </p>
+
+            <p><label for="password">Password:</label>
+              <input type="password" class="password" name="password" />
+            </p>
+
+            <p><label for="file">File Input:</label>
+              <input type="file" class="file" name="file" />
+            </p>
+
+
+            <p><input type="submit" value="Submit" /></p>
+          </form>
+        </fieldset>
+
+        <hr>
+
+        <h1 id="tables">Tables</h1>
+
+        <table cellspacing="0" cellpadding="0">
+          <tr>
+            <th>Table Header 1</th><th>Table Header 2</th><th>Table Header 3</th>
+          </tr>
+          <tr>
+            <td>Division 1</td><td>Division 2</td><td>Division 3</td>
+          </tr>
+          <tr class="even">
+            <td>Division 1</td><td>Division 2</td><td>Division 3</td>
+          </tr>
+          <tr>
+            <td>Division 1</td><td>Division 2</td><td>Division 3</td>
+          </tr>
+        </table>
+      </div>
+    </main>
+
+    <footer class="main">
+      <p>Bourbon is maintained and funded by <a href="//thoughtbot.com">thoughtbot, inc</a>. Tweet your questions or suggestions to <a href="//twitter.com/bourbonsass">@bourbonsass</a> and while you’re at it follow us too.</p>
```

Comment:
This information in the footer should be swapped for details of Bitters, instead of Bourbon.